lib/src/st2110/st_rx_audio_session.c

```diff
@@ -433,19 +440,24 @@ static int rx_audio_session_handle_rtp_pkt(struct mtl_main_impl* impl,
     }
   }
 
-  /* set first seq_id - 1 */
-  if (unlikely(s->latest_seq_id == -1)) s->latest_seq_id = seq_id - 1;
-  /* drop old packet */
-  if (st_rx_seq_drop(seq_id, s->latest_seq_id, 5)) {
+  if (unlikely(s->latest_seq_id[s_port] == -1)) s->latest_seq_id[s_port] = seq_id - 1;
+  if (seq_id != (uint16_t)(s->latest_seq_id[s_port] + 1)) {
+    dbg("%s(%d,%d), non-continuous seq now %u last %d\n", __func__, s->idx, s_port,
+        seq_id, s->latest_seq_id[s_port]);
+    ST_SESSION_STAT_INC(s, port_user_stats.common, stat_pkts_out_of_order);
+  }
+
+  /* Only works for redundant streams basicly drop packets that are old */
+  if (st_rx_seq_redundant_drop(seq_id, s->latest_seq_id, s_port, s->ops.num_port)) {
```

Comment:
can `s->ops.num_port` be only `2` if redundant mode is on, and `1` if the mode is off?

Reply:
yeeeees ?